internal/versions/api.go

```diff
@@ -29,17 +29,28 @@ type apiHandler struct {
 	log             logrus.FieldLogger
 	versionsHandler *handler
 	osImages        OSImages
+	releaseSources  models.ReleaseSources
+}
+
+type NewAPIHandlerParams struct {
```

Comment:
At this point why even have a `NewAPIHandler` function, just export apiHandler and have the caller set the fields directly.

Reply:
Reverted params struct